readme.md

```diff
@@ -39,64 +39,90 @@ $ npm -v
 * **Question:** What is the difference between forking and cloning a repository as opposed to just cloning a repository?
 
 * **Your Answer:**
+just cloning a repository - make no linking back to your github repo, no updaing when things are changed
+forking and cloning - linking to your github repo, updates can be pushed to online repo 
+
 
----
 
 - [ ] Run `npm init -y` from the command line
 
 * **Question:** What does `npm init` do? How does the `-y` flag modify that command?
 
 * **Your Answer:**
 
----
+npm init 
+to initialize a project
+promt for input from user
+project's name,
+project's version,
+project's description,
+
+npm init -y
+-y stands for yes
+generate an empty npm project without going through an interactive process
 
 - [ ] Take a look at the file that was generated by the previous command
 
 * **Question:** What is the purpose of the following keys? "name", "scripts", "license"
 
 * **Your Answer:**
 
----
+name: project's name
+scripts: to automate repetitive tasks, put them in scripts if you dont want to type long line into command promt
+license: project's license set to ISC Internet Systems Consortium license
 
 - [ ] Create a `.gitignore` file
 
 * **Question:** What is the purpose of the `.gitignore` file? What is the significance of a "dot-file?"
 
 * **Your Answer:**
 
----
+.gitignore
+contain files that to be ignore: pets.json
+not get push/update to GitHub
+files that change all the time while testing/building
+
+dot-file
+configuration files 
+hidden files
 
 - [ ] Create an `index.js` file with the following contents: `console.log('Hello, Node!')`
 
 * **Question:** From the command line, how can you run this file?
 
 * **Your Answer:**
 
----
+node index.js
 
 - [ ] Run `npm test` from the command line
 
 * **Question:** What happens and how is this related to what is in the `package.json` file? 
 
 * **Your Answer:**
 
----
+ "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
 
 - [ ] Create a new "script" command called "start" that has the following value: `node index.js`
 
 * **Question:** What will you enter on the command line to run that script?
 
 * **Your Answer:**
 
----
+npm start
+> w1-node-ecosystem@1.0.0 start C:\source\Summer2019\w1-node-ecosystem       
+> node index.js
+Hello, Node!
 
 - [ ] Change the name of your "start" script to "my-file"
 
 * **Question:** The same pattern will not work to try and run this script. How can you successfully get this script to run?
 
 * **Your Answer:**
 
----
+my-file is not one of npm command
```

Comment:
Correct! But then, how would you run it? In this case, you would type: `npm run my-file`